src/js/control/textarea.js

```diff
@@ -23,6 +23,8 @@ export default class controlTextarea extends control {
    */
   build() {
     const { value = '', ...attrs } = this.config
+    //Textareas do not have an attribute 'type'
+    delete attrs['type']
```

Comment:
We might not want a `delete` here because it will remove it completely from the scope and other parts of the project may require the `type` attribute as its used a lot. I think safer would be to destructure it out and not use it.
```suggestion
    const { value = '', type: _type, ...attrs } = this.config
    // Textareas do not have an attribute 'type'
```

Reply:
I thought about doing that, however the linter throws an no-unused-vars warning

```
WARNING in ../src/js/control/textarea.js
Module Warning (from ../node_modules/eslint-loader/dist/cjs.js):

../src/js/control/textarea.js
  26:31  warning  '_type' is assigned a value but never used  no-unused-vars

✖ 1 problem (0 errors, 1 warning)
```

I couldn't see where deleting type from the attrs array will have any downstream effect. The element's config property still contains the type value. attrs is only used for building the textarea element via this.markup

Reply:
ok lets leave it as-is for now then. We should add the following rule to the linter config:

```javascript
  'unused-imports/no-unused-vars': [
      'warn',
      {
        argsIgnorePattern: '^_',
        varsIgnorePattern: '^_',
      },
    ],
```